edn/src/edn.rustpeg

```diff
@@ -32,10 +32,10 @@ pub nil -> Value =
     "nil" { Value::Nil }
 
 pub nan -> Value =
-    "#f" __ "NaN" { Value::Float(OrderedFloat(NAN)) }
+    "#f " __ "NaN" { Value::Float(OrderedFloat(NAN)) }
 
 pub infinity -> Value =
-    "#f" __ s:$(sign) "Infinity" {
+    "#f " __ s:$(sign) "Infinity" {
```

Comment:
These are technically incorrect: EDN allows `','` as whitespace, so `#f,1.0` is valid (and you should add a test for this case). You want:

```
"#f" whitespace+ s:$(sign)
```

for this rule, and

```
"#f" whitespace+ "NaN"
```

for `nan`. That will also prevent users from putting a comment between a tag and its referent, which I think is a good thing!